packages/plugins/docs-ui/src/lib/components/cards/docs-cards.component.html

```
@@ -112,6 +112,22 @@
 				>
 					<nb-icon icon="external-link-outline"></nb-icon>
 				</button>
+				<!-- Row actions kebab (`01-ux-spec.md` §4.2: "kebab with the same row
+				     actions"), built by the same builder as the table column and the
+				     tree context menu. -->
+				<button
+					*ngIf="menuItemsFor(doc).length"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)